Cargo.lock

```diff
@@ -271,27 +180,12 @@ dependencies = [
  "winapi",
 ]
 
-[[package]]
-name = "async-sse"
-version = "4.1.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "53bba003996b8fd22245cd0c59b869ba764188ed435392cf2796d03b805ade10"
-dependencies = [
- "async-channel",
- "async-std",
- "http-types",
- "log",
- "memchr",
- "pin-project-lite 0.1.12",
-]
```

Comment:
It looks like async-std isn't gone — I suspect tls somewhere is to blame. Wdyt @pimeys ? Still a nice amount of removed dependencies, kudos!

Reply:
I was pinging @garrensmith already in Slack. I'm seeing if we can fix this with Cargo resolver v2, because it's Tiberius that does this.

Reply:
https://github.com/prisma/tiberius/pull/191

Reply:
resolved in [`f47d1be` (#2562)](https://github.com/prisma/prisma-engines/pull/2562/commits/f47d1be4efbcb9cb9114e18ea566b1b596305c19)